community/micro/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+export GOPATH="$PWD/go"
+export GO111MODULE=on
+
+# More verbose building.
+sed -i "s/go build/go build -v/" Makefile
```

Comment:
Should we care about this or just leave it default? I'm fine with it either way.